src/croniter/croniter.py

```diff
@@ -837,13 +837,18 @@ def is_leap(year):
     def value_alias(cls, val, field_index, len_expressions=UNIX_CRON_LEN):
         if isinstance(len_expressions, (list, dict, tuple, set)):
             len_expressions = len(len_expressions)
+
+        is_std_cron = len_expressions == UNIX_CRON_LEN
```

Comment:
well this one is not optimized as the unixcron is the most used , the other wont be evaluated if directly putted in the if statement.

Reply:
(so it will solve our readability problem but will consume more cycles ;))

Reply:
I'll just use `fmt off` for now haha.

I don't think we're going to lose a lot of CPU cylces from 2 extra boolean evaluations 😅

Reply:
Actually using a longer line length will fix this.

Reply:
> I don't think we're going to lose a lot of CPU cylces from 2 extra boolean evaluations 😅

As croniter is a generator, things can skid a lot quicker than scheduled :-).
I ve got bitten by this in the past.

That the purpose of this test case :  https://github.com/kiorky/croniter/blob/master/src/croniter/tests/test_speed.py

Reply:
(basically one user using `all_next` which consumes near whole iterator (https://github.com/kiorky/croniter/blob/master/src/croniter/croniter.py#L433) got a regression because of non optimized code.)